modules/ROOT/pages/data-routing.adoc

```diff
@@ -50,6 +50,40 @@ Creating a xref:sync-function-api.adoc[sync function] is a more flexible way to
 A sync function is a JavaScript function that takes a document body as input and, based on the document content, decides what channels to assign the document to.
 The sync function cannot reference any external state and must return the same results every time it's called on the same input.
 
+== Channel Count
+
+Sync Gateway can assign a document to new channels as long as the sync metadata remains under the allowed limit.
+
+The sync metadata contains the list of channel names to which a document is mapped.
+When a document is assigned to a new channel, the channel name is appended to that document's sync metadata.
+
+Therefore, the size that is occupied by channel sync metadata varies based on the following.
+
+* The channel name.
+A shorter channel name will occupy less space ("customer==0030169303" vs "cs==0030169303").
+* The value of revs_limit.
+Indeed, the sync metadata is retained for each non-leaf revision.
```

Comment:
```suggestion
Indeed, a copy of channel metadata is retained for each revision of a document.
```

We only have one sync metadata per document, but inside that there is one copy of channel metadata per revision.